### PR TITLE
Changelog for recent changes

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please (build details lost)

**Comment by Arsnael** [comment]
```
[e6b2f3b301336745fb800f5fe2faed9f4edb3c00] [ERROR] Errors: 
[e6b2f3b301336745fb800f5fe2faed9f4edb3c00] [ERROR]   MailsShouldBeWellReceived.mailsShouldBeWellReceived:63 » ConditionTimeout Cond...
```
test this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Add in-memory docker image
 - Support of AWS S3 as blobstore
+- WebAdmin routes for CLI capabilities on mapping management (AddAddressMapping, AddRegexMapping, ListMappings)
+
+### Changed
```
Comment (trantienduchn) [inline CHANGELOG.md]: I see 2 `Changed` sections. Could you merge into one?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- (experimental) Native implementation of the deleted message vault on top of an object store. This avoids exposing messages via webAdmin. Messages previously stored in the vault will be lost.
```
Comment (trantienduchn) [inline CHANGELOG.md]: why it's experimental?

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Add in-memory docker image
 - Support of AWS S3 as blobstore
+- WebAdmin routes for CLI capabilities on mapping management (AddAddressMapping, AddRegexMapping, ListMappings)
```
Comment (trantienduchn) [inline CHANGELOG.md]: - Vault metrics ?
 - Missing BlobStore metrics (deleteBucket & delete) ?

File: CHANGELOG.md (hunk 2 above)
Comment (chibenwa) [reply CHANGELOG.md]: Vault feature is experimental, isn't it?

Did we used it in production so far? Can we guaranty it will not break?

**Comment by chibenwa** [comment]
:green_apple: before CI reboot

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Metrics for Deleted Message Vault
+- More metrics for BlobStore (new delete & deleteBucket configuration)
 
 ### Changed
+- (experimental) Native implementation of the deleted message vault on top of an object store. This avoids exposing messages via webAdmin. Messages previously stored in the vault will be lost.
```
Comment (blackheaven) [inline CHANGELOG.md]: does it worth putting in the changelog if it's experimental?

Comment (chibenwa) [reply CHANGELOG.md]: IMO yes

Comment (blackheaven) [reply CHANGELOG.md]: so we'll have to go to the maintenance/deprecation process in case of change, that looks annoying for few benefits

Comment (chibenwa) [reply CHANGELOG.md]: Or we can remove the `(experimental)` from here...

Comment (blackheaven) [reply CHANGELOG.md]: I do not understand, if it's unstable, either we do not make it public, or we flag it as being unstable.

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Add in-memory docker image
 - Support of AWS S3 as blobstore
+- WebAdmin routes for CLI capabilities on mapping management (AddAddressMapping, AddRegexMapping, ListMappings)
```
Comment (mbaechler) [inline CHANGELOG.md]: I don't understand this sentence. How does it relate to CLI?

File: CHANGELOG.md (hunk 4 above)
Comment (mbaechler) [inline CHANGELOG.md]: What does Native mean?

Comment (rouazana) [reply CHANGELOG.md]: for me experimental is ok: you tell that there is a new feature, but also that it's experimental, so "use it at your very own risks"

File: CHANGELOG.md (hunk 5 above)
Comment (rouazana) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 4 above)
Comment (rouazana) [reply CHANGELOG.md]: fixed

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project metrics-api: Fatal error compiling: invalid flag: --release -> [Help 1]
[f842bc281c744d7a604f623a2dd6d6159fcba133] rogress (1): 6.5/13 MB
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project javax-mail-extension: Fatal error compiling: invalid flag: --release -> [Help 1]
```
Need a rebase... let me do it !

**Comment by chibenwa** [comment]
```
[1fe8dd87eed25d11533c9325bac3b3169b755e33] Location: /org/apache/james/imap/scripts/ValidateDeployment.test:499
[1fe8dd87eed25d11533c9325bac3b3169b755e33] LastClientMsg: c SEARCH FLAGGED
[1fe8dd87eed25d11533c9325bac3b3169b755e33] Expected: '\* SEARCH 1 2 3 4'
[1fe8dd87eed25d11533c9325bac3b3169b755e33] Actual   : '* SEARCH'
```

test this please

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**